src/web/modules/agreements/list.njk

```diff
@@ -0,0 +1,130 @@
+{% from "agreements/status.macro.njk" import status %}
+
+{% extends "layout/layout.njk" %}
+
+{% set isTestData = account and not (account.type === "live") %}
+
+{% set linkQueryParams %}{% if service %}&account={{ accountId }}{% endif %}{% if filters.reference %}&reference={{ filters.reference }}{% endif %}{% endset %}
+
+{% block main %}
+  <span class="govuk-caption-m">
+    {% if service %}
+      <span>{{ service.name }}</span>
+    {% else %}
+      <span>GOV.UK Pay platform</span>
+    {% endif %}
+  </span>
+
+  <h1 class="govuk-heading-m">Agreements</h1>
+
+  <div class="transactions-filter__container govuk-body">
+    {% for status in ["created", "active", "cancelled", "all"] %}
+      <a
+        class="transactions-filter__item {% if selectedStatus === status %}selected{% endif %} no-decoration"
+        href="/agreements?status={{ status }}{{ linkQueryParams }}">
+        <span>{{ status | capitalize }}</span>
+      </a>
+    {% endfor %}
+  </div>
+
+  {% if filters.email or filters.reference or filters.gateway_payout_id %}
```

Comment:
Agreements can't have an `email` or `gateway_payout_id`, I think a lot of this can be removed (we might want to filter by reference in the future so potentially that can stay?)

Reply:
Thanks